Terminal line selection

noahjg opened the pull request:
Solves the following issue:
Suggestion: Allow line selection in terminal #46761

Adds the ability to select next/previous lines in the integrated terminal by calculating the offset between the "cursor" and the selected line and using it to create a marker +/- 1 lines away from the currently selected line. Markers created this way are disposed whenever the current marker is changed, so the existing function of selectToNextCommand and selectToPreviousCommand is not compromised.

msftclas commented:
[![CLA assistant check](https://cla.opensource.microsoft.com/pull/badge/not_signed)](https://cla.opensource.microsoft.com/Microsoft/vscode?pullRequest=47991) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.microsoft.com/Microsoft/vscode?pullRequest=47991) before we can accept your contribution.<br/><br/>:x: noahjg [sign now](https://cla.opensource.microsoft.com/Microsoft/vscode?pullRequest=47991)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.microsoft.com/check/Microsoft/vscode?pullRequest=47991) it.</sub>

noahjg commented:
On precommit (and in the CI test) two of the files I modified the hygiene check:

File not formatted: src/vs/workbench/parts/terminal/node/terminalCommandTracker.ts
File not formatted: src/vs/workbench/parts/terminal/node/terminalCommandTracker.test.ts

However I cannot figure out why this is, as tslint doesn't find any errors and I followed all conventions stemming from the original part of the file I modified. How do I "format" these files that I have modified?

Tyriar commented:
@noahjg works great, just made a few cleanliness commits 👍

Tyriar commented:
Commands added in https://github.com/Microsoft/vscode/pull/47991
